### ISSUE TITLE
您好，代码109-115以及130行没有理解，能否帮忙解释一下这几行代码的意义？

### ISSUE THREAD

**Issue by Old-Lan** [RowitZou/DC-Match#4]
`kw_con_logits = self.kw_con_classifier(
            self.dropout(torch.cat([all_kw, sep_kw, all_con, sep_con], 0))
        )`
`kw_con_labels = torch.cat([labels.new_ones(all_kw.size(0) * 2),
                                   labels.new_zeros(all_con.size(0) * 2)], 0).float()`
`kw_con_loss = F.binary_cross_entropy_with_logits(kw_con_logits.view(-1), kw_con_labels.view(-1))`

**Comment by RowitZou** [comment]
这里对应的是论文的公式4，对keyword和其他词进行识别和分类。
